src/infra/services/servicesCmdRepo.go

```diff
@@ -248,18 +250,32 @@ func (repo *ServicesCmdRepo) createDefaultDirectories(
 			return errors.New("CreateDefaultDirsError: " + err.Error())
 		}
 
-		if execUser != nil {
-			execUserStr := execUser.String()
-			_, err := infraHelper.RunCmd("id", execUserStr)
-			if err != nil {
-				slog.Debug(err.Error(), slog.String("execUser", execUserStr))
-				continue
-			}
+		deletionWarningFilePath := defaultDirPath + "/DONOTDELETE"
+		_, err = os.Create(deletionWarningFilePath)
```

Comment:
Shouldn't you check if the directory exists first? I wouldn't throw an error if the file already exists.

Reply:
I don't check the directory because the `MakeDir` helper already handles this kind of error. In other words, if something went wrong during the directory creation, it would stop the process at `MakeDir`.
Either way, I think doing this extra validation is fine. It won't be redundant and will give me an additional guarantee with this double-check.